Formula/l/llvm@13.rb

```diff
@@ -23,6 +23,8 @@ class LlvmAT13 < Formula
 
   keg_only :versioned_formula
 
+  disable! date: "2024-12-31", because: :versioned_formula
```

Comment:
```suggestion
  deprecate! date: "2024-12-31", because: :versioned_formula
```

Reply:
Disabling for a future date means it's deprecated now, and then will be disabled on the date indicated, no?

Reply:
oh, that is true.